Classes/Swisscom/Neos/CaseInsensitiveUrls/Aspect/LowercaseRoutePartAspect.php

```diff
@@ -27,8 +27,14 @@ public function initializeObject() {
 	 * @return void
 	 */
 	public function convertRoutePartToLowerCase(\TYPO3\Flow\AOP\JoinPointInterface $joinPoint) {
-		$lowerCaseRequestPath = strtolower($joinPoint->getMethodArgument('requestPath'));
+		$requestPath = $joinPoint->getMethodArgument('requestPath');
 
+		if (strpos($requestPath, 'user')) {
```

Comment:
Usage of `user` is fault-prone as this could break all URLs with the string `user` in uriSegments.
@patrickreck: Why did you delete the at sign? This would make it more distinctive.